README.md

```diff
@@ -1,11 +1,12 @@
-# Build environment for Wigwag gateway firmware
+# Build environment for Pelion OS Edge firmware
```

Comment:
Pelion OS Edge?

This is the first time I've seen this branding.
CC @AnotherButler

Reply:
`Pelion Edge`, please.
@deepakpoornachandra to confirm